src/VisualStudio/Core/Test.Next/Services/SolutionAssetCacheTests.cs

```diff
@@ -20,6 +20,12 @@ namespace Roslyn.VisualStudio.Next.UnitTests.Remote
     [Trait(Traits.Feature, Traits.Features.RemoteHost)]
     public class SolutionAssetCacheTests
     {
+        private static void ForceGC()
+        {
+            for (var i = 0; i < 3; i++)
+                GC.Collect();
+        }
```

Comment:
this is a move of waht the product code used to do.  i moved it here so test code dosen't have to update.